Modules/Filtering/ImageFeature/include/itkHoughTransform2DCirclesImageFilter.hxx

```diff
@@ -32,7 +32,6 @@ HoughTransform2DCirclesImageFilter< TInputPixelType, TOutputPixelType, TRadiusPi
 ::HoughTransform2DCirclesImageFilter() :
 
   m_GradientNormThreshold{ 1.0 },
-
   m_UseImageSpacing{ true }
 
```

Comment:
... and another redundant extra line here! No problem, of course, I don't mind doing another HoughTransform2DCirclesImageFilter revision myself. Just wondering why clang-tidy did not move these two initializations to the class definition!

Reply:
Are these compiled by default?  Clang-tidy only addresses files that are part of the compilation scheme.

Reply:
@hjmjohnson Yes, `itkHoughTransform2DCirclesImageFilter` is compiled by default, and the file has been processed by your clang-tidy commit, b21dbbb2938911206d9259d3777b163d257031cd ("STYLE: Use default member initialization"). Apparently, clang-tidy skipped these two data members (`m_GradientNormThreshold` and `m_UseImageSpacing`) because they were already initialized in a _modern_ way (using curly braces). `:-)` I guess clang-tidy [modernize-use-default-member-init](https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-default-member-init.html) only does data members that were initialized using old-style (C++98) parentheses.

Reply:
Hi Hans! No I don't the ability to approve pull requests. But I guess I can fork a PR of yours, and present it as "my pull request", then you can approve it  ;-)

Reply:
@N-Dekker Thanks. Hans

Reply:
You're welcome. Just to be sure: My suggestion to fork your PR and then present it as mine wasn't meant to be serious. I think that would be a rather clumsy work flow!